packages/catalog-server/components/catalog/EndpointList.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Card, Button } from 'antd';
+import { ApiOutlined } from './utils';
+
+const StyledEndpointList = styled(Card)`
+  .endpointList-footer {
+    display: flex;
+    justify-content: flex-end;
+    flex-wrap: wrap;
+    * + * {
+      margin-left: 16px;
+    }
+  }
+  + .ant-card {
+    margin-top: 24px;
+  }
+`;
+
+const CardTitle = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: 16px;
+  font-weight: 700;
+  color: var(--gray-8);
+
+  .anticon {
+    margin-right: 8px;
+  }
+`;
+
+export interface EndpointListProps {
+  name: string;
+  description: string;
+  apiDocUrl: string;
+  onConnect: () => void;
+}
+
+export default function Endpoint(props: EndpointListProps) {
+  const {
+    name = 'test',
+    description = 'No description.',
```

Comment:
It will not show the `No description.` string if not set the description, since it is null.

If we want to show the `No description.` string, I prefer to use light gray to display it, so that the user will not misunderstand it.